### DISCUSSION TITLE
Safe to remove attachments from `/attachments/`?

### DISCUSSION THREAD

**Discussion by cuu508** [AsamK/signal-cli#1124] [Q&A] [answered]
I've recently added `--ignore-attachments` to the `signal-cli daemon` options. But I see there are a number attachments already received in the data directory, `/attachments/` subdirectory. Is it safe to remove them from the filesystem, or could it cause issues?

And, semi-related, I've also added `--no-receive-stdout` flag. Is there anything else I can / should do, to minimize the amount of received data that gets stored on the local system?

**Comment by AsamK** [comment, marked as answer]
Yes attachment files can be deleted anytime.

There's another option to prevent receiving story messages, if you're not interested in receiving those, but with the ignore attachments options those aren't stored anyway.
